src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -14,6 +14,12 @@ _ResolveAssemblies MSBuild target.
   <UsingTask TaskName="Xamarin.Android.Tasks.ProcessNativeLibraries" AssemblyFile="$(_XamarinAndroidBuildTasksAssembly)" />
   <UsingTask TaskName="Xamarin.Android.Tasks.StripNativeLibraries" AssemblyFile="$(_XamarinAndroidBuildTasksAssembly)" />
 
+  <PropertyGroup>
+    <AndroidEnableProfiler Condition=" '$(AndroidEnableProfiler)' == ''">false</AndroidEnableProfiler>
+    <AndroidEnableHotReload Condition=" '$(AndroidEnableHotReload)' == '' and '$(Configuration)' == 'Debug' ">true</AndroidEnableHotReload>
```

Comment:
now that I think about it do we really need a 'true` value for this? If this is controlled by the IDE then it will be responsible for passing in the value right? I which case we should just always default to `false` if `AndroidEnableHotReload` is not already set.

Reply:
Is hot reload usable outside the IDE?

Reply:
I dunno something to ask. It is a Dev Loop feature so I would guess it kinda needs an IDE.

Reply:
I'd also move these settings to `Microsoft.Android.Sdk.DefaultProperties.targets`.